### PR TITLE
#60 Fix Staging Style Error

### PR DESCRIPTION
**Pull request by michael-alade**
**Resolves**
#60

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "start": "npm run build && NODE_ENV=production node ./bin/server.js",
```
Comment (abiodun0) [inline package.json]: Change this to 
```"start": NODE_ENV=production node ./bin/server.js",```

**Comment by michael-alade** [comment]
@abiodun0 I just made changes

File: .travis.yml [hunk 2]
```diff
@@ -22,4 +22,4 @@ deploy:
     secure: o+Lhw3PDuV3DpuwQ64Ve9qfgMSME9dpcLlHXU3eO41n+5ZdnnovcOvaD1hKxrLAH1SGdbSgsqKHxkZsSBlNX7SSSEO2oQtfAPneUjyfsJ37Z8MvYY355Srm9kZ2yaUwQqyTvvZORz4fDqm1DNrnSZDdPSWKpfDGqzm12HHRmz8+aql9C8DQotro1PsD3pXKUD2sIBksUuJ+AMqj01xu+BHj+ppbSe+wqGxa9M3uAPL0JEw8MbVKO9np8SKz1HwDXt98Qn1jqbEQpISpjCB+nq9ILKjK0KNVDsEYzZePYsI5lxkgdknsX2DYW5yu1QvQnF6g+LmxQ4hS7tDE2WiEAftnGL+s2lKXo5W0Mity2Q9sDnZtxaeeEWSLeaJclpsMUjAJARcK2TIturK+lCS7XKautccvovzr0CaJlJanwbl+6eu02wvlqqY1XtoUA+M5ijrakQvdWMV7F3FK/PPaigQw+EpQShk1QA1MneAGY3GOX6xtiezzRBaQgDP1ZxP478vXGk8VMVuyfTPLUiKrjxcyPxlprJEbVeYsWfrYUSTit7P1Y+DbLC7e/pxKktPCHh1fdtyOAI20ss/BlxrR/ixzW7SPf9kJ5nv+IDicThveyBS1J63l3T4VVPIDN14gqArzS9lktJHyr1qCcgI6tSJgIGE8k6DEHKMNhk6bAy3M=
   app:
     master: taverna-webclient
-    staging: taverna-webclient-staging
+    test: team-taverna-testing
```
Comment (abiodun0) [inline .travis.yml]: When pushing to staging this should be reverted back to 
```taverna-webclient-staging```

Comment (michael-alade) [reply .travis.yml]: ok.

Comment (abiodun0) [reply .travis.yml]: and also update the branch to `staging`

Comment (michael-alade) [reply .travis.yml]: Do you mean rebase? @abiodun0

**Review by abiodun0 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**